.github/workflows/sonarcloud.io.yml

```diff
@@ -0,0 +1,50 @@
+name: SonarCloud.io
+on:
+   - pull_request
+   - push
+  
+jobs:
+  build:
+    name: SonarCloud.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Install Java 8
+        uses: AdoptOpenJDK/install-jdk@v1
+        with:
+          version: '8'
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.sonar/cache
+          key: ${{ runner.os }}-sonar
+          restore-keys: ${{ runner.os }}-sonar
+      - name: Cache Maven packages
+        uses: actions/cache@v1
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
+      - name: Maven Verify
+        run: cd app; mvn verify install -Psonarcloud -pl \!ui-react -fn                 
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Install Java 11
+        uses: AdoptOpenJDK/install-jdk@v1
+        with:
+          version: '11'
+      - name: Remove Problematic BOM project
+        run: rm -r app/extension/bom
+      - name: Remove Problematic BOM project reference
+        run: sed -i "/bom/d" app/extension/pom.xml
```

Comment:
Seems rather drastic, perhaps exclude via profile?

Reply:
The problem is that this project is inside the folder structure but it is not included inside the maven structure. And that's a huge bug for sonar (you can't even workaround it with "-fn")